test/unit/serialization_test.rb

```diff
@@ -2,9 +2,12 @@
 
 class Superstore::SerializationTest < Superstore::TestCase
   test 'as_json' do
-    issue = Issue.new
-    expected = {"id" => issue.id}
+    issue = Issue.new(skill_level_required: 100..200)
+    expected = {
+      "id" => issue.id,
+      "skill_level_required" => [100, 200]
```

Comment:
👎 

Override `Range#as_json` in the app. See how we override Time#as_json for reference.